temoa-uiuc/Snakefile

```diff
@@ -53,7 +53,7 @@ rule temoa_run:
         r"data_files/{db}_{scenario}_model/{scenario}_OutputLog.log",
         r"data_files/{db}_{scenario}_model/{scenario}.lp"
     shell:
-        "set +o pipefail; yes | python ~/research/temoa/temoa_model --config={input}"
+        "set +o pipefail; yes | python $TEMOA/temoa_model --config={input}"
```

Comment:
```suggestion
        "set +o pipefail; yes | python ~/research/temoa_model --config={input}"
```
I'm not sure what $TEMOA is... it might be better but for now it should be ~/research/temoa because that's what the instructions in the README say.

Reply:
your commit missed the /temoa/

you can add these lines to your ~/.bashrc file:

export TEMOA=$HOME/projects/temoa/

which in your case should be export TEMOA=$HOME/research/temoa/

and then you can call it with $TEMOA from anywhere.

Reply:
Got it, I will add instructions to the README!